Hermes/appscale/hermes/__init__.py

```diff
@@ -12,30 +9,31 @@
 import socket
 import sys
 import tarfile
+import urllib
+
+import argparse
 import tornado.escape
 import tornado.httpclient
 import tornado.web
-import urllib
-
 from appscale.common import appscale_info
 from appscale.common import appscale_utils
+from appscale.common.constants import LOG_FORMAT
 from appscale.common.ua_client import UAClient
 from appscale.common.ua_client import UAException
-from handlers import MainHandler
-from handlers import TaskHandler
-from helper import JSONTags
-
 from tornado.ioloop import IOLoop
 from tornado.ioloop import PeriodicCallback
-from tornado.options import define
 from tornado.options import options
-from tornado.options import parse_command_line
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '../AppServer'))
+import helper
+from appscale.hermes.constants import DEFAULT_HERMES_PORT
```

Comment:
[PEP8#imports](https://www.python.org/dev/peps/pep-0008/#imports) accepts explicit relative imports, but it recommends to use absolute imports if packages layout is not too complex.